TYPO3.Media/Classes/TYPO3/Media/ViewHelpers/ImageViewHelper.php

```diff
@@ -104,14 +104,14 @@ public function initializeArguments()
         $this->registerTagAttribute('alt', 'string', 'Specifies an alternate text for an image', true);
         $this->registerTagAttribute('ismap', 'string', 'Specifies an image as a server-side image-map. Rarely used. Look at usemap instead', false);
         $this->registerTagAttribute('usemap', 'string', 'Specifies an image as a client-side image-map', false);
-        // @deprecated since 2.0 use the "image" argument instead
-        $this->registerArgument('asset', 'TYPO3\Media\Domain\Model\AssetInterface', 'The image to be rendered - DEPRECATED, use "image" argument instead', false);
+        // @deprecated since 2.1 use "asset" argument instead
+        $this->registerArgument('image', 'TYPO3\Media\Domain\Model\ImageInterface', 'The image to be rendered - DEPRECATED, use "asset" argument instead', false);
```

Comment:
> For users there's no difference, it's all backwards compatible.

Right now yes, but we deprecate something that we deprecated, so worst case scenario is someone adapted the codebase after the 2.0 upgrade to use "image" instead of "asset" and now they need to do the same again back to "asset". I would be pretty annoyed about it.